contextualSpellCheck/contextualSpellCheck.py

```diff
@@ -243,7 +260,7 @@ def misspell_identify(self, doc, query=""):
         # deep copy is required to preserve individual token info
         # from objects in pipeline which can modify token info
         # like merge_entities
-        docCopy = copy.deepcopy(doc)
+        docCopy = doc  # copy.deepcopy(doc)
```

Comment:
Deepcopy is required as mentioned in the comment above. Any specific reason to drop it?

Reply:
Hi, as per my profiling, deepcopy caused a lion's share of time spent. When I disabled it, in my particular case nothing changed except the execution speed. Maybe it's possible to recognize situations where it's needed automatically, and only enable deepcopy for such runs? Or have some optional flag to allow faster processing of big texts that need no merge_entities etc.

Reply:
I understand. It was required to address https://github.com/R1j1t/contextualSpellCheck/issues/9. Maybe for now, we can keep the deep copy and try to optimize in a different PR? This PR already has 3 major changes!

Reply:
Ok, sure.